lib/TransportRPCServer.js

```diff
@@ -3,6 +3,12 @@
 const TransportRPC = require('./TransportRPC')
 
 class TransportRPCServer extends TransportRPC {
+  constructor (client, conf, Socket) {
+    super(client, conf)
+
+    this.Socket = Socket
```

Comment:
haha i had that before, removed it this afternoon, as you pointed me to `PeerRPC.js` this morning so i thought i must have missed something

after thinking about it again you are right, and it should also solve the problem described in the other comment